DESCRIPTION

```diff
@@ -16,7 +16,7 @@ URL: https://github.com/open-systems-pharmacology/ospsuite-r
 BugReports: 
     https://github.com/open-systems-pharmacology/ospsuite-r/issues
 Depends: 
-    R (>= 3.5),
+    R (>= 3.6),

```

Comment:
Why? It works fine under 3.5 - what is the reason to require 3.6?

Reply:
the readme says it works under 3.6 :) I assumed this was accurate?!?

Reply:
seem to remember some issue with rclr and 3.5. But maybe I am wrong

Reply:
I don't think this is going to work here as we need different version of rclr based on the version of R

Reply:
see https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation

Reply:
we install the 4.x version specifically.

Reply:
here https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/appveyor.yml#L40

Reply:
Oh, I see! Then we should wait and ascertain that the CI indeed fails :)

Okay, outside the CI context, I am not really sure how to check if the package works with `3.5`.

Reply:
ah ah yes!

Looks like at least it will nicely mess up the version number lol
![image](https://user-images.githubusercontent.com/1041237/134225588-e279b006-fa1f-4845-a6f1-138e5cc868bf.png)